src/vsphere_cpi/spec/integration/nsxt_spec.rb

```diff
@@ -407,6 +407,54 @@
     end
   end
 
+  describe 'on adding a vm to security groups' do
+    let(:vm_type) do
+      {
+        'ram' => 512,
+        'disk' => 2048,
+        'cpu' => 1,
+        'nsxt' => nsxt_spec
+      }
+    end
+    let(:nsxt_spec) do
```

Comment:
Just put it back up in vm_type above. it is a single line and you are not modifying it in any examples below.

Reply:
Done.